translations/en.json

```diff
@@ -1203,6 +1203,7 @@
     "repeat-puk": "Repeat new 12-digit PUK",
     "report-bug-email-template": "1. Issue Description\n{{description}}\n\n\n2. Steps to reproduce\n{{steps}}\n\n\n3. Attach screenshots that can demo the problem, please\n",
     "request-processed-after-node-online": "Your request will be processed once the community owner node is back online.",
+    "request-to-join-disclaimer": "Joining the community will reveal your public addresses to the owner",
```

Comment:
looking at the designs, I think this might only need to be shown if the community is token gated. Can you verify this?

Reply:
The PR's `Notes` section has all the context related to this point.

> [Figma > Mobile App > Communities for Mobile](https://www.figma.com/file/h9wo4GipgZURbqqr1vShFN/Communities-for-Mobile?type=design&node-id=1733-125697&t=3ssa500fwGvUqiTL-0) is outdated and will be changed soon, but what's important for this PR is that the final disclaimer Joining the community will reveal your public addresses to the node owner must always be shown, with/without token gated communities.
>
>After this PR I will create a separate issue related to changes that will be needed to make the Request to Join button work with upcoming changes in status-go that the desktop team is working on.

Reply:
Issue created https://github.com/status-im/status-mobile/issues/16196